.releaserc

```diff
@@ -100,6 +100,18 @@
       }
     ],
     "@semantic-release/changelog",
+    [
+      "@semantic-release/npm",
+      {
+        "npmPublish": false
+      }
+    ],
+    [
+      "@semantic-release/exec",
```

Comment:
You need to install the package

Reply:
Done. Could you re-review? @ilfa

Reply:
We also need to release `v1.0.1`. Do you want me to create an empty commit? I can also change commit messages with `fix` prefix, and force push. @ilfa

Reply:
let's rewrite the commit message